test/cpp/interop/interop_client.h

```diff
@@ -154,6 +167,9 @@ class InteropClient {
   ServiceStub serviceStub_;
   /// If true, abort() is not called for transient failures
   bool do_not_abort_on_transient_failures_;
+  // Load Orca metrics captured by the custom LB policy. Needs to be initialized
```

Comment:
I don't think we need the "Needs to be initialized first" comment anymore.

Reply:
I didn't mean to remove the whole comment, just the second sentence.

Reply:
Done